config/initializers/devise.rb

```diff
@@ -317,4 +317,19 @@
     approval_prompt: 'select_account consent force',
     scope: 'userinfo.email,userinfo.profile,calendar'
   )
+
+  require 'omniauth-facebook'
+  config.omniauth(
+    :facebook,
+    ENV['FACEBOOK_CLIENT_ID'],
+    ENV['FACEBOOK_CLIENT_SECRET'],
+    scope: 'public_profile,email',
+    info_fields: 'email,first_name,last_name',
+    client_options: {
+    site: 'https://graph.facebook.com/v2.11',
```

Comment:
If this site changes based on the Version number for graph, please make it an ENV value instead.

Reply:
@bmwachajr sorry I didn't get you. What should I change to ENV value?
I currently have this:
`ENV['FACEBOOK_APP_ID'],
 ENV['FACEBOOK_APP_SECRET']
`